builtin/game/privileges.lua

```diff
@@ -86,3 +86,8 @@ core.register_privilege("debug", {
 	description = "Allows enabling various debug options that may affect gameplay",
 	give_to_singleplayer = false,
 })
+
+core.register_on_userlimit_check(function(name, ip)
+	local privs = core.get_player_privs(name)
+	return (privs["server"] or privs["ban"] or privs["privs"] or privs["password"])
```

Comment:
Why did you use square brackets and quotes instead of *.* here?

Reply:
parenthesis are optional yes, but quoted are mandatory, the dict index is textual